index.html

```diff
@@ -0,0 +1,317 @@
+            please reach out!
+          </p>
+          <a
+            href="https://docs.google.com/document/d/1lYq0Kb50TujzJzDWGobIchRI17JMnAiZMF9OBQgNU68/edit?usp=sharing"
+            class="link"
+            target="_blank"
+            rel="noreferrer noopener"
+          >
+            Get my resume
+          </a>
+          <img class="about-me-icon" src="./media/svg/icon-2.svg" alt="Icon" />
+        </section>
+        <hr />
+        <div class="about-me-section stacks">
+          <ul>
+            <li class="languages list-title">
+              Languages
+              <ul>
+                <li>JavaScript</li>
+                <li>TypeScript</li>
+                <li>Ruby</li>
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>Sass</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="frameworks list-title">
+              Frameworks
+              <ul>
+                <li>React & Redux</li>
+                <li>Material-UI</li>
+                <li>Jest</li>
+                <li>Bootstrap</li>
+                <li>Ruby on Rails</li>
+                <li>RSPec</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="skills list-title">
+              Skills
+              <ul>
+                <li>GitHub</li>
+                <li>CodePen</li>
+                <li>GitLab</li>
+                <li>Terminal</li>
+                <li>WSL</li>
+                <li>Remote Pair-Programming</li>
+                <li>Teamwork</li>
+                <li>Mentoring</li>
+                <li>Code Reviews</li>
+                <li>Mobile/Responsive Development</li>
+              </ul>
+            </li>
+          </ul>
+        </div>
+      </div>
+
+      <footer>
+        <h2 id="contact-text">I'm always interested n hearing about new projects so if you'd like to chat please get in</h2> 
+        <form action="https://formspree.io/f/meqvlald" method="POST">
+            <input class="contact-input" type="text" id="name" name="name" maxlength="30" placeholder="Full name" required>
+            <input class="contact-input" type="email" id="email" placeholder="Email address" name="_replyto" required>
+            <textarea class="contact-input" name="message" id="message"  maxlength="500"
+              placeholder="Enter text here" required></textarea>
+          <button id="submit-btn" type="submit">Get in Touch</button>
+        </form>
+        <ul id="footer-icons">
+          <li><img src="./media/svg/github.svg" alt="git-icon"/></li>
+          <li><img src="./media/svg/linkedin.svg" alt="linkedin-icon"/></li>
+          <li><img src="./media/svg/angel.svg" alt="angel-icon"/></li>
+          <li><img src="./media/svg/twitter.svg" alt="twitter-icon"/></li>
+          <li><img src="./media/svg/medium.svg" alt="medium-icon"/></li>
+        </ul>
+      </footer>
+    </main>
```

Comment:
[OPTIONAL] Since the previous reviewer did not point this out, the following will be marked as optional in this PR.

Good job on using HTML5 semantic tags!

However, semantically, the <footer> should be outside the <main> element. This will help to more accurately describe the relationship between HTML elements in your code.

It is also recommended to add blank lines to separate large or logical code blocks, for readability. It would be good to add a blank line around the contact form section and the social icons section.